5_recognition.ipynb

```diff
@@ -238,25 +425,50 @@
     "HSVは、色相(Hue)、彩度(Saturation)、明度(Value)によって色を表現します。今回は色相情報のみを利用してレゴブロックの水色を抽出します。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "The color of the Lego block is \"light blue\", so let's use the color to detect the Lego block.\n",
+    "\n",
+    "Color data is stored in RGB format in \"image_data\".\n",
+    "We can use the RGB data as is, but it is known that converting the color space from RGB to HSV makes it more robust against lighting changes.\n",
```

Comment:
"We can use the RGB data as is, but it is known that converting the color space from RGB to HSV makes it more robust against changes in lighting.\n",